### PR TITLE
ov 2.0: python speech_recognition_deepspeech_demo

### PR DESCRIPTION
**Pull request by anzhella-pankratova**
Migrate python speech_recognition_deepspeech_demo to OV2.0 API.
**The demo does not work, because the model's input/output nodes names are changed by original tensorflow names.** So, the profile file will contain different names for old and new API. Waiting for updates from MO side solving the issue of models nodes naming

### CODE REVIEW

**Comment by eaidova** [comment]
@anzhella-pankratova 

> The demo does not work, because the model's input/output nodes names are changed by original tensorflow names. (CVS-75841)

We do not need to preserve demo support for both old and new API in the same time, so I think it is not big problem to change model output names, especially if they aligned with original framework

**Comment by anzhella-pankratova** [comment]
@eaidova We would do it, if both models mozilla-deepspeech-0.6.1 and mozilla-deepspeech-0.8.2 were contain only tensorflow names for nodes. But mozilla-deepspeech-0.6.1 has both old and new names, allowing launch the demo with old and new API correspondently. In this case I believe to modify the names for the second model is a better option and we can wait for update. 

Alternately, I can replace the names in the demo in order to run tests. If the tests are passed, we will merge the PR, and I will provide the other PR to support both API when the conversion for mozilla-deepspeech-0.8.2 is fixed. @vladimir-dudnik what do you think?

**Comment by vladimir-dudnik** [comment]
@anzhella-pankratova when you say "mozilla-deepspeech-0.6.1 has both old and new names, allowing launch the demo with old and new API" you mean IR v10 and IR v11?

**Comment by anzhella-pankratova** [comment]
> @anzhella-pankratova when you say "mozilla-deepspeech-0.6.1 has both old and new names, allowing launch the demo with old and new API" you mean IR v10 and IR v11?

Yes. For example, for IR v10 we have "logits" name, for IR v11 "logits:0". For mozilla-deepspeech-0.6.1 the both these names can be accessed, but for mozilla-deepspeech-0.8.2 the input name contains only "logits:0" name

**Comment by vladimir-dudnik** [comment]
> > @anzhella-pankratova when you say "mozilla-deepspeech-0.6.1 has both old and new names, allowing launch the demo with old and new API" you mean IR v10 and IR v11?
> 
> @vladimir-dudnik Yes. For example, for IR v10 we have "logits" name, for IR v11 "logits:0". For mozilla-deepspeech-0.6.1 the both these names can be accessed, but for mozilla-deepspeech-0.8.2 the input name contains only "logits:0" name

@andrei-kochin it seems like MO issue ? We do not expect changes in names between IR v10 and IR v11 ?.

**Comment by ilyachur** [comment]
> > > @anzhella-pankratova when you say "mozilla-deepspeech-0.6.1 has both old and new names, allowing launch the demo with old and new API" you mean IR v10 and IR v11?
> > 
> > 
> > @vladimir-dudnik Yes. For example, for IR v10 we have "logits" name, for IR v11 "logits:0". For mozilla-deepspeech-0.6.1 the both these names can be accessed, but for mozilla-deepspeech-0.8.2 the input name contains only "logits:0" name
> 
> @andrei-kochin @ilya-lavrenov @ilyachur it seems like MO or runtime issue ? We do not expect changes in names between IR v10 and IR v11 ?.

Why don't you expect changes in names after migration to v11?
In IR v11 we are using tensor names, in IR v10 operation names. It means that we are using different names and they can be different.

**Comment by vladimir-dudnik** [comment]
@ilyachur basically you say, 2022.1 applications will have complexities if they have to support 2021.4.2 IRs.

**Comment by vladimir-dudnik** [comment]
@anzhella-pankratova we will generate intel models with IR v11 soon (hope on this week), you do not need to support IR v10 names in 2022.1 application (if one need this there is always 2021.4.2 release available)

**Comment by anzhella-pankratova** [comment]
> @anzhella-pankratova we will generate intel models with IR v11 soon (hope on this week), you do not need to support IR v10 names in 2022.1 application (if one need this there is always 2021.4.2 release available)

@vladimir-dudnik Okay, then I will update the names.

**Comment by eaidova** [comment]
looks like tests, passed, but need to add sys.exit for avoid issue with memory deallocation on python api side

**Review by eaidova — APPROVED** [review]
